include/deal.II/grid/tria_accessor.h

```diff
@@ -2489,6 +2496,116 @@ class CellAccessor :  public TriaAccessor<dim,dim,spacedim>
+   * we can make sure that the element on the other side of the periodic
+   * boundary exists in this rank as a ghost cell or a locally owned cell.
+   */
+  TriaIterator<CellAccessor<dim, spacedim> >
+  periodic_neighbor (const unsigned int i) const;
+
+  /**
+   * Returns an iterator to the periodic neighbor of the cell at a given
+   * face and subface number. The general guidelines for using this function
+   * is similar to the function @c neighbor_child_on_subface. The
+   * implementation of this function is consistent with
+   * @c periodic_neighbor_of_coarser_periodic_neighbor. For instance,
+   * assume that we are sitting on a cell named @c cell1, which has a 1 level
+   * coarser neighbor at its @c ith face. Let us name this coarser neighbor
+   * @c cell2. Then, by calling
+   * @c periodic_neighbor_of_coarser_periodic_neighbor, from @c cell1, we get
+   * a @c face_num and a @c subface_num. Now, if we call
+   * @c periodic_neighbor_child_on_subface from cell2, with the above face_num
+   * and subface_num, we get an iterator to @c cell1.
+   */
+  TriaIterator<CellAccessor<dim, spacedim> >
+  periodic_neighbor_child_on_subface (const unsigned int face_no,
+                                      const unsigned int subface_no) const;
+
+  /**
+   * This function is a generalization of
+   * @c periodic_neighbor_of_periodic_neighbor
+   * for those cells which have a coarser periodic neighbor. The returned
+   * pair of numbers can be used in @c periodic_neighbor_child_on_subface
+   * to get back to the current cell. In other words, the following
+   * assertion should be true, for a cell with coarser periodic neighbor:
+   * cell->periodic_neighbor(i)->periodic_neighbor_child_on_subface(face_no, subface_no)==cell
+   */
+  std::pair<unsigned int, unsigned int>
+  periodic_neighbor_of_coarser_periodic_neighbor (const unsigned i) const;
+
+  /**
+   * This function returns the index of the periodic neighbor. If there is
+   * no periodic neighbor at the given face, the returned value is -1.
+   */
+  int
+  periodic_neighbor_index (const unsigned int i) const;
+
+  /**
+   * This function returns the level of the periodic neighbor. If there is
+   * no periodic neighbor at the given face, the returned value is -1.
+   */
+  int
+  periodic_neighbor_level (const unsigned int i) const;
+
+  /**
+   * For a cell with a periodic neighbor at its @c ith face, this function
+   * returns the face number of that periodic neighbor such that, the
+   * current cell is the periodic neighbor of that neighbor. In other words
+   * the following assertion holds for those cells which have a periodic
+   * neighbor with the same or a higher level of refinement as the current
+   * cell:
+   * @c {cell->periodic_neighbor(i)->
+   *     periodic_neighbor(cell->periodic_neighbor_of_periodic_neighbor(i))==cell}
+   * For the cells with a coarser periodic neighbor, one should use
+   * @c periodic_neighbor_of_coarser_periodic_neighbor and
```

Comment:
Instead of using `@c` here, just say `periodic_neighbor_of_coarser_periodic_neighbor()` and doxygen will use the parentheses as an indication that this is a function that can be linked to. (It will then also use the appropriate font.)